src/content/en/tools/chrome-devtools/javascript/get-started.md

```diff
@@ -0,0 +1,274 @@
+
+If you're using `console.log()` to find and fix bugs in your code, consider
+using the workflow outlined in this tutorial instead. It's often much faster
+and more effective.
+
+## Step 1: Reproduce the bug {: #step-1 }
+
+Reproducing the bug is always the first step to debugging.
+To "reproduce the bug" means to find a series of actions that consistently
+causes the bug to appear. You may need to reproduce the bug many times,
+so try to eliminate any unnecessary steps.
+
+Follow along with the instructions below to reproduce the bug that you're going
+to fix in this tutorial.
+
+1. Click **Open Demo**. The demo opens in a new tab.
+
+     <a href="https://googlechrome.github.io/devtools-samples/debug-js/get-started"
+       target="devtools"
+       rel="noopener noreferrer">
+       <button>Open Demo</button>
+     </a>
+
+1. On the demo, enter `5` for **Number 1**.
+1. Enter `1` for **Number 2**.
+1. Click **Add Number 1 and Number 2**.
+1. Look at the label below the inputs and button. It says `5 + 1 = 51`.
+
+Whoops. That result is wrong. The result should be `6`. This is the bug that
+you're going to fix.
+
+## Step 2: Pause the code with a breakpoint
+
+DevTools lets you pause your code in the middle of its execution, and
+examine the values of *all* variables at that moment in time. The tool for
+pausing your code is called a **breakpoint**. Try it now:
+
+1. Open DevTools on the demo by pressing
+   <kbd>Command</kbd>+<kbd>Option</kbd>+<kbd>I</kbd> (Mac) or
+   <kbd>Control</kbd>+<kbd>Shift</kbd>+<kbd>I</kbd> (Windows, Linux).
+1. Click the **Sources** tab.
+1. Click **Event Listener Breakpoints** to expand the section. DevTools reveals
+   a list of expandable event categories, such as **Animation** and
+   **Clipboard**.
+1. Next to the **Mouse** event category, click **Expand** ![Expand
+   icon](/web/tools/chrome-devtools/images/expand.png){: .devtools-inline}.
+   DevTools reveals a list of mouse events, such as **click**,
+   with checkboxes next to them.
+1. Check the **click** checkbox.
+1. Back on the demo, click **Add Number 1 and Number 2** again. DevTools
+   pauses the demo and highlights a line of code in the **Sources** panel.
+1. Check that DevTools is highlighting this line of code:
+
+       `function onClick() {`
+
+1. If it's highlighting any other line, keep pressing **Resume script
+   execution** ![Resume script execution][resume]{: .devtools-inline }
+   until DevTools highlights the line of code above. The highlighted line
+   represents the line of code that you're currently paused on.
+
+When you checked the **click** checkbox, you set up an event-based breakpoint on
+all `click` events. When *any* node is clicked, and that node has a `click`
+handler, DevTools automatically pauses on the first line of that node's
+`click` handler.
+
+Note: This is just one of the many types of breakpoints that DevTools offers.
+What breakpoint you should use depends on what type of issue you're debugging.
```

Comment:
So, my reason for not linking to the breakpoints doc here is, I want people to complete the tutorial. Having a link here invites the reader to open that link and get distracted by the other doc, reducing their chances of completing this tutorial. Thoughts?